cfgov/jinja2/v1/_includes/molecules/featured-content.html

```diff
@@ -9,43 +9,59 @@
+   value.body:            Body text.
 
-   value.post:           Page being featured.
+   value.show_post_link:  Boolean flag for whether to render the post link.
 
-   value.show_post_link: Flag to render the post link
+   value.post:            (Optional) Page being featured.
 
-   value.post_link_text: String for the link text of the displayed post
+   value.post_link_text:  String for the link text of the displayed post.
 
-   value.macro_link:     If the FCM is rendered outside of wagtail,
-                         this is a simpler way of adding a link.
-                         This is an object made up of url and text.
+   value.macro_link:      (Optional) Object used if the FCM is
+                          rendered outside of wagtail,
+                          this is a simpler way of adding a link.
 
-   value.links:          If the FCM is rendered in wagtail,
-                         this returns the links.
+   value.macro_link.url:  String for URL of the link.
 
-   value.video:          This object gets passed to video-player.html
-                         By default, the following values should exist:
-                           'is_fcm': true,
-                           'height': '320',
-                           'width':  '568'
-                         Wagtail should set the following values:
-                           'id', 'url'
+   value.macro_link.text: String for text of the link.
 
-   value.image:          A image object containing the URL & alt text for screenreaders
-                         of the image to be placed adjacent to the text.
+   value.links:           If the FCM is rendered in wagtail,
+                          this returns the array of links.
 
-   value.links:          A collection of links with URL & Text.
+   value.links[i].url:    String for URL of the link.
+
+   value.links[i].text:   String for text of the link.
+
+   value.video:           (Optional)
+                          This object gets passed to macros/video-player.html.
+                          By default, the following values should exist:
+                            'is_fcm': true,
+                            'height': '320',
+                            'width':  '568'
+                          Wagtail should set the following values:
+                            'id', 'url'
+
+   value.video.url:       A String used to denote the url of the video.
+
+   value.video.id:        A String used to denote the id of the video.
+
+   value.image:           (Optional) An object containing details
+                          of the image to be placed adjacent to the text.
+
+   value.image.upload:    An image object.
+
+   value.image.alt:       String for alt text for screenreaders.
 
    ========================================================================== #}
 
 {% macro render(value) %}
 {% import 'macros/category-icon.html' as category_icon %}
 
+{# TODO: Update classes to use m- molecule prefix. #}
```

Comment:
Updated in https://github.com/cfpb/cfgov-refresh/pull/2032/commits/249ef825fb1582dc31e850b66e1b0506e9629d37